src/ResourceManager/Kubernetes/AzureRM.Kubernetes.psd1

```diff
@@ -0,0 +1,124 @@
+#
+# Module manifest for module 'PSGet_AzureRM.Kubernetes'
+#
+# Generated by: Microsoft Corporation
+#
+# Generated on: 11/3/2017
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+# RootModule = ''
+
+# Version number of this module.
+ModuleVersion = '0.0.1'
+
+# Supported PSEditions
+# CompatiblePSEditions = @()
+
+# ID used to uniquely identify this module
+GUID = 'a97e0c3e-e389-46a6-b73d-2b9bd6909bdb'
+
+# Author of this module
+Author = 'Microsoft Corporation'
+
+# Company or vendor of this module
+CompanyName = 'Microsoft Corporation'
+
+# Copyright statement for this module
+Copyright = 'Microsoft Corporation. All rights reserved.'
+
+# Description of the functionality provided by this module
+Description = 'Microsoft Azure PowerShell - - Azure managed Kubernetes commands'
+
+# Minimum version of the Windows PowerShell engine required by this module
+PowerShellVersion = '3.0'
+
+# Name of the Windows PowerShell host required by this module
+# PowerShellHostName = ''
+
+# Minimum version of the Windows PowerShell host required by this module
+# PowerShellHostVersion = ''
+
+# Minimum version of Microsoft .NET Framework required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+DotNetFrameworkVersion = '4.5.2'
+
+# Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+CLRVersion = '4.0'
+
+# Processor architecture (None, X86, Amd64) required by this module
+# ProcessorArchitecture = ''
+
+# Modules that must be imported into the global environment prior to importing this module
+RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '4.0.0'; })
+
+# Assemblies that must be loaded prior to importing this module
+RequiredAssemblies = '.\YamlDotNet.dll'
```

Comment:
All assemblies required by the cmdlet assembly should go here, normally this includes the associated management library.

Reply:
The generated management library is included in the command module. This will need to be removed and replaced with a reference when the SDK is produced.